recipes/lzo/all/conanfile.py

```diff
@@ -12,7 +12,7 @@ class LZOConan(ConanFile):
     license = "GPL-v2.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://www.oberhumer.com/opensource/lzo/"
-    topics = ("lzo", "compression")
+    topics = ("compression")
```

Comment:
it's an invalid tuple

Reply:
easy to make such a mistake. is it possible to linter? @ericLemanissier @uilianries @prince-chrismc

Reply:
Base class ConanFile should anotate the type of this member as tuple of strings, then mypy should be able to flag this error